notebooks/how_to/run_unit_metrics.ipynb

```diff
@@ -4,41 +4,141 @@
     "- They quantify risk and can be used to monitor and assess risks associated with a model's entire lifecycle.\n",
     "- Measurable, relevant, and linked to risk areas and critical business processes - e.g., regulatory requirements, risk appetite, model performance, data quality.\n",
     "- Standalone in nature, meaning they do not rely on other metrics for their calculation or interpretation.\n",
     "\n",
-    "Incorporating Unit Metrics into your ML workflow streamlines risk assessment, turning complex analyses into clear, actionable insights.\n"
+    "Incorporating unit metrics into your ML workflow streamlines risk assessment, turning complex analyses into clear, actionable insights.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Contents\n",
+    "- [About ValidMind](#toc1_)    \n",
+    "  - [Before you begin](#toc1_1_)    \n",
+    "  - [New to ValidMind?](#toc1_2_)    \n",
+    "  - [Key concepts](#toc1_3_)    \n",
+    "- [Initialize the client library](#toc2_)    \n",
+    "- [Notebook setup](#toc3_)    \n",
+    "- [Load the demo dataset](#toc4_)    \n",
+    "- [Train a model for testing](#toc5_)    \n",
+    "- [Compute predictions](#toc6_)    \n",
+    "- [Initialize ValidMind objects](#toc7_)    \n",
+    "- [Assign predictions](#toc8_)    \n",
+    "- [Running unit metrics](#toc9_)    \n",
+    "  - [Compute F1 score](#toc9_1_)    \n",
+    "  - [Access metric results](#toc9_2_)    \n",
+    "  - [Pass parameters](#toc9_3_)    \n",
+    "  - [Load the last computed value](#toc9_4_)    \n",
+    "  - [Unit metrics for model performance](#toc9_5_)    \n",
+    "  - [Run multiple unit metrics as a single test](#toc9_6_)    \n",
+    "  - [Adding composite metrics to the documentation template](#toc9_7_)    \n",
+    "    - [Reconnect to the documentation project](#toc9_7_1_)    \n",
+    "- [Next steps](#toc10_)    \n",
+    "  - [Work with your model documentation](#toc10_1_)    \n",
+    "  - [Discover more learning resources](#toc10_2_)    \n",
+    "\n",
+    "<!-- vscode-jupyter-toc-config\n",
+    "\tnumbering=false\n",
+    "\tanchor=true\n",
+    "\tflat=false\n",
+    "\tminLevel=2\n",
+    "\tmaxLevel=4\n",
+    "\t/vscode-jupyter-toc-config -->\n",
+    "<!-- THIS CELL WILL BE REPLACED ON TOC UPDATE. DO NOT WRITE YOUR TEXT IN THIS CELL -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='toc1_'></a>\n\n",
+    "## About ValidMind\n",
+    "\n",
+    "ValidMind is a platform for managing model risk, including risk associated with AI and statistical models. You use the ValidMind Developer Framework to automate documentation and validation tests, and then use the ValidMind AI Risk Platform UI to collaborate on model documentation. Together, these products simplify model risk management, facilitate compliance with regulations and institutional standards, and enhance collaboration between yourself and model validators.\n",
```

Comment:
```suggestion
        "ValidMind is a platform for managing model risk, including risk associated with AI and statistical models. \n",
        "\n",
        "You use the ValidMind Developer Framework to automate documentation and validation tests, and then use the ValidMind AI Risk Platform UI to collaborate on model documentation. Together, these products simplify model risk management, facilitate compliance with regulations and institutional standards, and enhance collaboration between yourself and model validators.\n",
```

Reply:
I agree with this edit, but multi-line replaces in notebooks can be problematic. Changed globally to `ValidMind is a platform for managing model risk, including risk associated with AI and statistical models.\n\nYou ...` for now to add the linebreaks.